resources/views/yourstore.blade.php

```diff
@@ -52,11 +52,9 @@
 		                </form>
 		                @endif
 		                @if(Auth::check() && $user->id == Auth::user()->id)
-						<form  action="/ad/{{ $product->id }}" method="POST">
-			                {{ csrf_field() }}
-			                <input type="hidden" name="_method" value="DELETE">
-			                <button id="deleteWishlistBtn" type="submit" class="close" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-		                </form>
+		                <button type="button" class="close" aria-label="Close" data-toggle="modal" data-target="#deleteAdModal" data-productid="{{ $product->id }}">
+		                	<span aria-hidden="true">&times;</span>
```

Comment:
I have been getting rid of the  ARIA labels (aria-hidden, aria-visible etc...) its for reader only, I find it clutters up he code too mich. Any thoughts? @BrandonBalala